tests/translator/input/all_policy_templates.yaml

```diff
@@ -170,7 +170,7 @@ Resources:
           EventBusName: name
 
       - AcmGetCertificatePolicy:
-          CertificateArn: arn
+          CertificateArn: arn:aws:acm:us-west-2:987654321098:certificate/dec86919-7219-4e8d-8871-7f1609df2c7f
```

Comment:
Is it a random arn or it comes from somewhere?

Reply:
random arn since this is just checking if the transform is correct